docs/code-howtos/tools.md

```diff
@@ -37,7 +37,7 @@ See also: [https://help.github.com/articles/syncing-a-fork/](https://help.github
 (As Administrator - one time)
 
 1. Install [chocolatey](https://chocolatey.org)
-2. `choco install git.install -y --params "/GitAndUnixToolsOnPath /WindowsTerminal"`
+2. `choco install git.install -y --params "/GitAndUnixToolsOnPath /WindowsTerminal`
```

Comment:
are you sure about that? That does not seem right...

Reply:
I am sorry i am not aware when i changed the file , must have been a mistake

Reply:
no need to be sorry, mistakes happen and we learn. as a general rule, always check your changes before committing them. its easy if you use "git gui" to commit instead of the command line or the integrazes tools in the ide.